int32 num_pizza_loaded
float32 x_pos
float32 y_pos
